modules/gathering-data-specific-features.adoc

```diff
@@ -28,10 +28,10 @@ ifndef::openshift-origin[]
 |`registry.redhat.io/container-native-virtualization/cnv-must-gather-rhel8:v{HCOVersion}`
 |Data collection for {VirtProductName}.
 
-|`registry.redhat.io/openshift-serverless-1/svls-must-gather-rhel8`
+|`registry.redhat.io/openshift-serverless-1/svls-must-gather-rhel8:2.2`
 |Data collection for OpenShift Serverless.
 
-|`registry.redhat.io/openshift-service-mesh/istio-must-gather-rhel8`
+|`registry.redhat.io/openshift-service-mesh/istio-must-gather-rhel8:2.2`
```

Comment:
why not use the latest, i.e. 2.3?

Reply:
@jwendell that's a good point. The story was created when 2.2 was newer so that might be why. I just updated the PR.